packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -191,22 +188,82 @@ export interface ContainerRuntimeMessage {
     contents: any;
     type: ContainerMessageType;
 }
+export interface ISummaryConfigurationBaseSettings {
```

Comment:
Consider `ISummaryBaseConfiguration`. Having both configuration and settings in the name is redundant.